api/query/atoms.go

```diff
@@ -188,9 +185,7 @@ type AbstractNone struct {
 // nolint:ireturn // TODO: Fix ireturn lint error
 func (e AbstractNone) BindToRuns(runs ...shared.TestRun) ConcreteQuery {
 	return Not{
-		AbstractExists{
-			Args: e.Args,
-		}.BindToRuns(runs...),
+		AbstractExists(e).BindToRuns(runs...),
```

Comment:
Question: Why did this change from 
```
AbstractExists{
			Args: e.Args,
		}.
```

to 

`AbstractExists(e)`

Reply:
This was flagged by gosimple S1016 - replacing manual copy of struct fields with type conversions

Reply:
Thanks for this! I actually learned something from S1016